src/controllers/users-controller.ts

```diff
@@ -35,6 +37,103 @@ export default class UsersController extends AbstractController {
     return this.getEntityById<User>(id);
   }
 
+  @PUT
+  @Path('/:id')
+  @Response<{}>(200, 'User successfully updated')
+  @Response<CustomError>(401, 'Unauthorised')
+  public async updateUser(
+    body: any,
+    @PathParam('id') id: string,
+    @ContextRequest req: Request & { user: User },
+  ): Promise<{} | void> {
+    if (id !== req.user.id) {
+      throw new CustomError('Unauthorized', 401);
+    }
+    const { firstName, lastName } = body as User;
```

Comment:
Also we should update users phone

Reply:
Well.... But at the moment the model does not provide user's phone
I have no problem adding that, do you have some kind of flow with phones?